src/time.cr

```diff
@@ -331,6 +331,12 @@ struct Time
     now(Location::UTC)
   end
 
+  # Returns a `Tuple` with `year`, `month` and `day`.
+  def date : Tuple(Int32, Int32, Int32)
+    year, month, day, _ = year_month_day_day_year
```

Comment:
You don't need the `_` here

Reply:
Don't need it but I think it's preferable to have it..

Reply:
I don't see why.. maybe only for style: to show that there are other values we don't care about

Reply:
Exactly.